api/src/api/opportunities_v1/opportunity_routes.py

```diff
@@ -212,3 +212,28 @@ def opportunity_versions_get(db_session: db.Session, opportunity_id: int) -> res
         data = get_opportunity_versions(db_session, opportunity_id)
 
     return response.ApiResponse(message="Success", data=data)
+
+
+@opportunity_blueprint.post("/user/token")
+@opportunity_blueprint.output(opportunity_schemas.OpportunityUserTokenResponseV1Schema)
+@opportunity_blueprint.auth_required(api_key_auth)
```

Comment:
This should be in a separate blueprint from opportunities.

I'd recommend we create an `auth` folder / blueprint / etc. instead.

Reply:
Added the auth folder with the blueprint, routes and schema files

Reply:
I'd make a folder called users (similar to `opportunities_v1`). The blueprint you create should have the prefix `/v1/users` - the route would then be `/v1/users/token` when all put together.